unified-runtime/CMakeLists.txt

```diff
@@ -137,8 +137,16 @@ if(UR_ENABLE_TRACING)
     add_compile_definitions(UR_ENABLE_TRACING)
 
     if (UR_BUILD_XPTI_LIBS)
-        # fetch xpti proxy library for the tracing layer
-        FetchContentSparse_Declare(xpti https://github.com/intel/llvm.git "nightly-2024-10-22" "xpti")
+        set(UR_XPTI_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../xpti")
```

Comment:
If we're doing a standalone build that happens to be in an LLVM directory (or any project that happens to also have an xpti folder), do we want to pick it up?

I think if we are building a standalone build (presumably by setting `-DUR_STANDALONE_BUILD` or something), that should be the source of truth as to whether we pull in the parent directory's xpti dir or fetch it remotely.

Reply:
We don't have a CMake flag for a standalone build, simply not existing inside the intel/llvm source tree is considered a standalone build.

Reply:
Remember, [oneapi-src/unified-runtime](https://github.com/oneapi-src/unified-runtime) can still be cloned separately and that is the primary consideration for standalone builds.